loglist2/loglist2_test.go

```diff
@@ -435,7 +440,8 @@ func TestLogStatesString(t *testing.T) {
 		want   string
 	}{
 		{name: "ReadOnly", logURL: "https://ct.googleapis.com/aviator/", want: "ReadOnlyLogStatus"},
-		{name: "Empty", logURL: "https://ct.googleapis.com/icarus/", want: "UndefinedLogStatus"},
+		{name: "Empty", logURL: "https://ct.googleapis.com/racketeer/", want: "UndefinedLogStatus"},
```

Comment:
_Did you mean: r**o**cketeer?_ :)

Reply:
I mean 'rAcketeer'. Our sample Log-list is interesting reading

Reply:
Alrighty, then :)